apps/backend/src/modules/meetings/meetings.controller.ts

```diff
@@ -0,0 +1,286 @@
+ *         ownerId:
+ *           type: number
+ *     MeetingCreateRequest:
+ *       type: object
+ *       required:
+ *         - host
+ *         - ownerId
+ *       properties:
+ *         host:
+ *           type: string
+ *         instanceId:
+ *           type: string
+ *           nullable: true
+ *         ownerId:
+ *           type: number
+ *     MeetingUpdateRequest:
+ *       type: object
+ *       required:
+ *         - host
+ *         - instanceId
+ *         - ownerId
+ *       properties:
+ *         host:
+ *           type: string
+ *         instanceId:
+ *           type: string
+ *           nullable: true
+ *         ownerId:
+ *           type: number
+ */
+
+class MeetingsController extends BaseController {
+	private meetingRepository: MeetingRepository;
+
+	public constructor(logger: Logger, meetingRepository: MeetingRepository) {
+		super(logger, APIPath.MEETINGS);
+		this.meetingRepository = meetingRepository;
+
+		this.addRoute({
+			handler: (options) =>
+				this.findAll(options as APIHandlerOptions & { user: { id: number } }),
+			method: "GET",
+			path: MeetingsApiPath.ROOT,
+		});
+
+		this.addRoute({
+			handler: (options) =>
+				this.find(
+					options as APIHandlerOptions<{ params: { id: string } }> & {
+						user: { id: number };
+					},
+				),
+			method: "GET",
+			path: MeetingsApiPath.$ID,
+		});
+
+		this.addRoute({
+			handler: (options) =>
+				this.create(
+					options as APIHandlerOptions<{ body: MeetingCreateRequestDto }> & {
+						user: { id: number };
+					},
+				),
+			method: "POST",
+			path: MeetingsApiPath.ROOT,
+			validation: { body: meetingCreateValidationSchema },
+		});
+
+		this.addRoute({
+			handler: (options) =>
+				this.update(
+					options as APIHandlerOptions<{
+						body: MeetingUpdateRequestDto;
+						params: { id: string };
+					}> & {
+						user: { id: number };
+					},
+				),
+			method: "PATCH",
+			path: MeetingsApiPath.$ID,
+			validation: { body: meetingUpdateValidationSchema },
+		});
+
+		this.addRoute({
+			handler: (options) =>
+				this.delete(
+					options as APIHandlerOptions<{ params: { id: string } }> & {
+						user: { id: number };
+					},
+				),
+			method: "DELETE",
+			path: MeetingsApiPath.$ID,
+		});
+	}
+
+	/**
+	 * @swagger
+	 * /meetings:
+	 *   post:
+	 *     summary: Create a new meeting
+	 *     requestBody:
+	 *       required: true
+	 *       content:
+	 *         application/json:
+	 *           schema:
+	 *             $ref: "#/components/schemas/MeetingCreateRequest"
+	 *     responses:
+	 *       201:
+	 *         description: Meeting created
+	 *         content:
+	 *           application/json:
+	 *             schema:
+	 *               $ref: "#/components/schemas/Meeting"
+	 */
+	private async create(
+		options: APIHandlerOptions<{ body: MeetingCreateRequestDto }> & {
+			user: { id: number };
+		},
+	): Promise<APIHandlerResponse> {
+		const service = new MeetingService(this.meetingRepository, options.user.id);
```

Comment:
You can't initialize a service in controller. It should be initialized inside meetings.ts. For checking the user id, add fastify hook